src/bootstrap/builder/tests.rs

```diff
@@ -8,9 +8,9 @@ fn configure(cmd: &str, host: &[&str], target: &[&str]) -> Config {
     config.save_toolstates = None;
     config.dry_run = true;
     config.ninja_in_file = false;
-    config.out = PathBuf::from(env::var_os("BOOTSTRAP_OUTPUT_DIRECTORY").unwrap());
-    config.initial_rustc = PathBuf::from(env::var_os("RUSTC").unwrap());
-    config.initial_cargo = PathBuf::from(env::var_os("BOOTSTRAP_INITIAL_CARGO").unwrap());
+    config.out = PathBuf::from(env!("OUT_DIR"));
+    config.initial_rustc = PathBuf::from(env!("RUSTC"));
+    config.initial_cargo = PathBuf::from(env!("CARGO"));
```

Comment:
IIRC these are sometimes relative paths unless that was fixed?

In any case, it seems like initial_rustc / initial_cargo here are identical to what config.rs would set anyway so we can probably just skip setting them here?

Reply:
> IIRC these are sometimes relative paths unless that was fixed?

We make them absolute in the build script - I would expect that to also be true for tests? Not 100% sure how cargo sets the variables.

> In any case, it seems like initial_rustc / initial_cargo here are identical to what config.rs would set anyway so we can probably just skip setting them here?

Ah yes, good point, I'll remove them :)